OpenLanding cleaning up

fiskus opened the pull request:
* Remove marketing blocks
* Update stats numbers
* Enlarge the search block
* ~~Use custom Intercom launcher~~
* Disable Intercom

- [x] [Changelog](../tree/master/docs/CHANGELOG.md) entry (skip if change is not significant to end users, e.g. docs only)

![Screenshot from 2025-03-24 15-34-25](https://github.com/user-attachments/assets/c2991431-bfbc-41c0-84dd-4acafddd7eb3)

nl0 commented on catalog/app/website/pages/OpenLanding/Search/Search.js:
```diff
@@ -167,11 +167,11 @@ export default function Search() {
           <div className={classes.stats}>
             <div className={classes.stat}>
-              <div className={classes.statValue}>10.2 Billion</div>
+              <div className={classes.statValue}>477 Million</div>
               <div className={classes.statDesc}>Objects</div>
```
```suggestion
              <div className={classes.statValue}>Millions</div>
              <div className={classes.statDesc}>Of Objects</div>
```

nl0 commented on catalog/app/website/pages/OpenLanding/Search/Search.js:
```diff
@@ -167,11 +167,11 @@ export default function Search() {
             </div>
             <div className={classes.stat}>
-              <div className={classes.statValue}>3.7 Petabytes</div>
+              <div className={classes.statValue}>1.3 Petabytes</div>
```
```suggestion
              <div className={classes.statValue}>Petabytes</div>
```
let's keep this opaque for now

nl0 commented:
tho i don't quite like the transition between the search section and the bucket grid (there's none).
i think it would be nice to add some (maybe put some kind of background under the bucket grid? idk)

fiskus commented:
@nl0 What about this?
* I removed stats and made the search bar bigger
* Added divider and semi-opaque background for buckets grid
also, added some little animations

nl0 commented:
i like it, tho i would probably adjust some spacing:
1. less space above the filter box (maybe make it the same as below the box)
2. more space between the background image and the bucket grid, for the transition to feel somewhat less abrupt

<img width="808" alt="Screenshot 2025-03-25 at 12 15 00" src="https://github.com/user-attachments/assets/752c964f-c9d5-4baf-88da-7a5a6a7a64d8" />


up to you tho.

@QuiltSimon @kevinemoore @drernie ?

drernie commented:
Approved.

fiskus commented:
I'll be back to polishing it if there will be some spare time
